roles/openshift_master_facts/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+openshift_master_default_subdomain: "{{ lookup('oo_option', 'openshift_master_default_subdomain') | default(None, true) }}"
```

Comment:
Required after pulling in https://github.com/openshift/openshift-ansible/pull/2908 to have `openshift_master_default_subdomain` set in upgrade (for `openshift_master_facts` role). We're [setting this in a few places as a workaround](https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/redeploy-certificates.yml#L55-L62) so I've added to defaults so that it's always there.